### PR TITLE
I6099 : Support section in review box issue

### LINKED ISSUE #6099: "support section" from review redirects to addon details page
Steps to reproduce:

1. Load AMO homepage https://addons.mozilla.org/en-US/firefox/ .
2. Click on any addon.
3. Click on the "Write a Review" button.
4. Click on the "support section" hyperlink.

Expected results:
The user is redirected to the "Support Site".

Actual results:
The user is redirected to the addon details page.

Notes/Issues:
Verified on FF54(Win 10). Issue is reproducing on all AMO servers.
Gif for this issue:
support from write review

### PR DESCRIPTION
**Pull request by tsl143**
Fixes #6099 
(Refer hyperlinks in the screenshots)
**Before:**
screen shot 2017-09-07 at 9 14 23 pm

**After:**
screen shot 2017-09-07 at 9 10 24 pm

### CODE REVIEW

File: src/olympia/addons/templates/addons/impala/review_add_box.html [hunk 1]
```diff
@@ -30,12 +30,14 @@
...
-            {% trans %}
-              See this add-on's <a href="{{ support }}">support section</a> to find out if assistance is available.
-              You can also try asking the <a href="https://discourse.mozilla-community.org/c/add-ons/add-on-support">add-on community</a> for help.
-            {% endtrans %}
-          {% endwith %}
+          See this add-on's 
+          {% if addon.support_url %}
+            <a class="support" href="{{ addon.support_url|external_url }}">{{ _('support section') }}</a>
+          {% elif addon.support_email %}
+            {{ emaillink(addon.support_email.localized_string, _('support E-mail')) }}
```
Comment (EnTeQuAk) [inline src/olympia/addons/templates/addons/impala/review_add_box.html]: Should be `Support Email` imho, or `support email` but not with the dash

**Comment by tsl143** [comment]
@EnTeQuAk , I will go with small case so as to keep text in sync with `support section`

**Comment by EnTeQuAk** [comment]
Thanks a lot for your contribution, I merged this and it should go online next Thursday.

**Comment by tsl143** [comment]
Awesome :)

**Pull request merged**